synapse/models/language.py

```diff
@@ -0,0 +1,47 @@
+
+def getDataModel():
+    return {
+
+        'prefix':'lang',
+
+        'types':(
+            #('lang:code',{'subof','enum',...
+            ('lang:idiom',{'subof':'str:txt','doc':'A subcultural idiom'}),
+            ('lang:trans',{'subof':'str:txt','doc':'Raw text with a documented translation'}),
```

Comment:
Do we want something like lang:gist for "unofficial" (e.g., Google-translate type) translations? We def want "official" lang:trans ones, but in the absence of a native speaker it would still be nice to store an approximation if only to save having to go out and Google-translate it anyway.